pkg/api/flow.go

```diff
@@ -4432,6 +4442,159 @@ func (h *flowHandler) BroadcastCloudevent(w http.ResponseWriter, r *http.Request
 
 }
 
+func (h *flowHandler) BroadcastCloudeventFilter(w http.ResponseWriter, r *http.Request) {
+
+	h.logger.Debugf("Handling request: %s", this())
+
+	ctx := r.Context()
+	namespace := mux.Vars(r)["ns"]
+	filter := mux.Vars(r)["filter"]
+
+	ces, err := ToGRPCCloudEvents(r)
+	if err != nil {
+		respond(w, nil, err)
+		fmt.Println(err)
```

Comment:
This printline and others like it is debug/devlopment code, which should be removed.